ConsoleApp/Program.cs

```diff
@@ -0,0 +1,110 @@
+﻿class Program
+{
+    static int MaxValue(int firstParam, int secondParam)
+    {       
+        if (firstParam > secondParam) return firstParam;
+        if (firstParam < secondParam) return secondParam;
+        else return firstParam;        
```

Comment:
else not needed here, just `if (firstParam > secondParam) return firstParam; return secondParam;`